app/assets/javascripts/map/helpers/guides.js

```diff
@@ -159,7 +146,7 @@ define([
 
       // Layers module
       {
-        selector: "#layersnav-forest-change", 
```

Comment:
We need to use the same editor config to avoid this kind of diffs (Mine is config to avoid trailing spaces)